src/js/components/List/List.js

```diff
@@ -245,7 +247,19 @@ const List = React.forwardRef(
             as={as || 'ul'}
             itemFocus={itemFocus}
             tabIndex={onClickItem || onOrder ? 0 : undefined}
-            onBlur={onOrder ? () => setActive(undefined) : undefined}
+            onBlur={
+              onOrder
```

Comment:
I think we should always `setLastActive` in `onBlur`, irrespective of whether `onOrder` is being used.

Reply:
Updated so now it doesn't have to check for `onOrder`, works as intended still.